pkg/sshclient/bastion.go

```diff
@@ -0,0 +1,174 @@
+package sshclient
+
+import (
+	"bufio"
+	"fmt"
+	"io/ioutil"
+	"net"
+	"net/url"
+	"os"
+	"path/filepath"
+	"strconv"
+	"time"
+
+	"github.com/containers/storage/pkg/homedir"
```

Comment:
Is there a way to avoid this import? It brings a lot of source code just for reading some env. variables.

Reply:
yes this can be avoided, will remove

Reply:
removed